### ISSUE TITLE
Github Authentication Fails on Branch Push

### ISSUE THREAD

**Issue by zacheryph** [gitbutlerapp/gitbutler#2760]
I have authenticated against Github. The Project Settings gives me the green checkmark. However, Everytime I attempt to push a branch up, I get an authorization error. I told Project Settings to use the Git Credentials Helper, and I still get an error. `git push` works totally fine...

**Comment by krlvi** [comment]
Hey, sorry for the troubles. I added some context in your other GH issue https://github.com/gitbutlerapp/gitbutler/issues/2762 regarding the `refs/gitbutler` containing the refs.
Did you manage to get pushing working correctly with any of the authentication options? If not I can help debug (and if it's convenient for you, you can ping me on discord)

**Comment by Byron** [comment]
GitButler Version 0.11.3 (20240419.070253) has also gained the option to authenticate with the Git binary, and overall authentication was improved a lot.

Is there a chance you could retry with [the latest version](https://gitbutler.com)?

**Comment by Qix-** [comment]
Indeed, will close for now - please use the system executable auth mechanism in your project settings. It should be the default from now on for new projects, too.

<img width="623" alt="grafik" src="https://github.com/gitbutlerapp/gitbutler/assets/885648/123dddca-84bf-454b-8a7c-a4352d5a304e">

Let me know if this somehow doesn't work for you and I'll re-open :) Thanks for the report!
